assignment08/index.php

```diff
@@ -0,0 +1,29 @@
+
+<?php
+    include "includes/header.php";
+    $my_weight= '125 lb';
+    $_age = 25;
+    $first_name='Donjeta';
+    $last_name='Emerllahu';
+    $poundsToKg= 'pounds *(1.2.2)';
+    //write PHP here...
+   function poundsToKg($my_weight){
+       $my_weight = ("pounds *(1/2.2");
+       return $my_weight;
+    
+   }
+   $result = add ($my_weight + $poundsToKg)
```

Comment:
$my_weight should be sufficient for outputting your weight in pounds. You don't need this.